jennchenn/uwexplore
# Added filtering menu

## Implementation description

- Clicking on the filtering button now opens the filtering menu
- Checkboxes correspond to diff. filtering options
- Can close the menu or apply the filters through buttons
- Resolves BREAD-22



## Steps to test

1. Click filtering button -> does the menu open?
2. Apply filters, and check that each value is logged in the console
3. Cancel applying filters, and verify that the menu closes



## What should reviewers focus on?

- Will we be implementing all of these filtering options?
